packages/containers/.storybook/config.js

```diff
@@ -57,13 +57,18 @@ api.expression.register('modelHasLabel', context => {
 function loadStories() {
 	Object.keys(examples).forEach(example => {
 		const state = mock.state();
+		const value = new Map({ id: 'myID', label: 'myLabel' });
+		const collections = new Map({
+			myResourceType: new List([value]),
+		});
+		state.cmf.collections = collections;
```

Comment:
Now state has collections you can do the following 
state.cmf.collections = state.cmf.collections.set('myResourceType', new List([value]))